### PR TITLE
Add `publish_on_sync` config option

### PR DESCRIPTION
**Pull request by stankovicjovana**
Name is now required field for every collection of WebFlow.
Re-record all cassettes to be up to date with the WebFlow API.
Added new cassette when `publish_on_sync` is set to `true`.

### CODE REVIEW

File: spec/jobs/webflow_sync/create_item_job_spec.rb [hunk 1]
```diff
@@ -91,5 +98,24 @@ module WebflowSync
...
+        end
+      end
+
+      after(:each) do
+        WebflowSync.configure do |config|
+          config.publish_to_all_domains = @publish_to_all_domains
+        end
+      end
+
+      it 'publishes all domains', vcr: { cassette_name: 'webflow/create_item_and_publish', record: :once } do
```
Comment (vfonic) [inline spec/jobs/webflow_sync/create_item_job_spec.rb:116]: Remove `record: :once` here

Comment (vfonic) [reply spec/jobs/webflow_sync/create_item_job_spec.rb:116]: How do we know this publishes all domains?

We could set the test here to check if `publish` method was called or not.

File: spec/jobs/webflow_sync/update_item_job_spec.rb [hunk 2]
```diff
@@ -5,9 +5,16 @@
...
   RSpec.describe WebflowSync::UpdateItemJob, type: :job do
     let(:collection_slug) { 'articles' }
-    let(:article) { create(:article, webflow_item_id: 'webflow_item_id') }
+    let(:article) { create(:article, webflow_item_id: '60defde681813e53c6be97ea') }
     let(:mock_webflow_api) { instance_double('mock_webflow_api', update_item: nil) }
 
+    before(:each) do
+      WebflowSync.configure do |config|
+        config.webflow_site_id = 'webflow_site_id'
+        config.api_token = 'webflow_api_token'
```
Comment (vfonic) [inline spec/jobs/webflow_sync/update_item_job_spec.rb:14]: If we'd get these using `ENV.fetch`, then we wouldn't have to specify them like this.

**Review by vfonic — CHANGES_REQUESTED** [review]

File: spec/jobs/webflow_sync/update_item_job_spec.rb (hunk 2 above)
Comment (stankovicjovana) [reply spec/jobs/webflow_sync/update_item_job_spec.rb]: Thank you! Installed `dotenv-rails`, and start using` ENV.fetch`.

File: README.md [hunk 3]
```diff
@@ -42,8 +42,9 @@ In `config/initializers/webflow_sync.rb` you can specify configuration options:
 
 1. `api_token`
 2. `webflow_site_id`
-3. `skip_webflow_sync` - skip synchronization for different environments
-4. `sync_webflow_slug` - save slug generated on WebFlow to the Rails database, in the Rails model column. 
+3. `skip_webflow_sync` - skip synchronization for different environments.
+4. `publish_to_all_domains` - republish all domains after create and update actions. Republish is called after every delete action by default.
```
[the remark this answers is not in the document]
Comment (stankovicjovana) [reply README.md]: Renamed. Made it the same for all actions. Thanks :)

**Pull request merged**